modulemd/tests/test-modulemd-moduleindex.c

```diff
@@ -540,9 +458,9 @@ module_index_test_stream_upgrade (void)
     MODULEMD_MODULE_STREAM_V2 (stream), "A second stream's description");
   modulemd_module_stream_v2_add_module_license (
     MODULEMD_MODULE_STREAM_V2 (stream), "Beerware");
-  ret = modulemd_module_index_add_module_stream (index, stream, &error);
+  g_assert_true (
+    modulemd_module_index_add_module_stream (index, stream, &error));
   g_assert_no_error (error);
-  g_assert_true (ret);
```

Comment:
\<sigh>It would be nice if all my tedious work to assert on error messages before return codes could be kept...\</sigh>

Reply:
It was supposed to be; auto-merge-resolution appears to have been wrong.